From 2f683c0279b5e1def4d8e9f2a45ae721c0fe014c Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster) <raster@rasterman.com>
Date: Sun, 2 Nov 2008 12:04:20 +1100
Subject: [PATCH] e-wm: fix nice level set for e

can't use it with exec as its ONLY a shell builtin with busybox. use renice
---
 packages/e17/e-wm/enlightenment_start.oe |    3 ++-
 packages/e17/e-wm_svn.bb                 |    2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/packages/e17/e-wm/enlightenment_start.oe b/packages/e17/e-wm/enlightenment_start.oe
index fa78302..e5a6502 100644
--- a/packages/e17/e-wm/enlightenment_start.oe
+++ b/packages/e17/e-wm/enlightenment_start.oe
@@ -5,4 +5,5 @@ if [ -r /etc/enlightenment/default_profile ]; then
   . /etc/enlightenment/default_profile
 fi
 
-exec nice -n -10 enlightenment_start $E_PROFILE
+renice -10 -p $$
+exec enlightenment_start $E_PROFILE
diff --git a/packages/e17/e-wm_svn.bb b/packages/e17/e-wm_svn.bb
index a641a2c..a2fabc7 100644
--- a/packages/e17/e-wm_svn.bb
+++ b/packages/e17/e-wm_svn.bb
@@ -2,7 +2,7 @@ DESCRIPTION = "The Enlightenment Window Manager Version 17"
 DEPENDS = "eet evas ecore edje efreet edbus"
 LICENSE = "MIT BSD"
 PV = "0.16.999.043+svnr${SRCREV}"
-PR = "r16"
+PR = "r17"
 
 inherit e update-alternatives
 
-- 
1.5.4.3

